src/plugins/saved_search/public/services/saved_searches/get_saved_searches.ts

```diff
@@ -20,6 +21,7 @@ interface GetSavedSearchDependencies {
   search: DataPublicPluginStart['search'];
   savedObjectsClient: SavedObjectsStart['client'];
   spaces?: SpacesApi;
+  savedObjectsTagging: SavedObjectsTaggingApi | undefined;
```

Comment:
Nit: 🎨 For consistency with` spaces?`, this could be 
```suggestion
  savedObjectsTagging?: SavedObjectsTaggingApi;
```